dct_backend/reviews.py

```diff
@@ -0,0 +1,62 @@
+from fastapi import FastAPI, Response, status
+from dotenv import load_dotenv
+import os
+from database import Database
+import server
+from fastapi.testclient import TestClient
+
+load_dotenv()
+db = Database(os.getenv("DB_USERNAME"), os.getenv("DB_PASSWORD"))
+db.connect()
+
+food_db = db.client["food"] # from food collection
+review_collection = food_db["reviews"] # review --> sub collection
+app = FastAPI()
+resp = Response
+
+class Reviews():
+    @app.post("/reviews", status_code=status.HTTP_201_CREATED)
+    async def save_review_in_db(review_rating: dict, is_test = False):
```

Comment:
You use dicts, that's fine, but we want to remain consistent with the "Base Model" they use. Feel free to take a peak in #143, it uses the updated refactorization from #105 so don't confuse yourself too much. You're looking for the `User` class